src/Xamarin.Android.Build.Tasks/Properties/Resources.resx

```diff
@@ -655,6 +655,10 @@ In this message, the term "bundled" is a short way of saying "included into the
     <comment>The following are literal names and should not be translated: Wear, APK, SignAndroidPackage
 "Wear" is a short version of the full product name "Wear OS" and so should not be translated.</comment>
   </data>
+  <data name="XA4312" xml:space="preserve">
+    <value>Embedding an Android Wear application inside an Android application is deprecated. Distribute the Wear application as a standalone application instead.</value>
+    <comment>The following are literal names and should not be translated: Android, Wear.</comment>
```

Comment:
For the sake of being extra explicit to minimize any questions the localization might have, I'll suggest including "Android Wear" as a literal name too:

```
<comment>The following are literal names and should not be translated: Android Wear, Android, Wear.</comment>
```